src/db/DB_Table_Subcategory_V1.h

```diff
@@ -100,60 +99,6 @@ struct DB_Table_SUBCATEGORY_V1 : public DB_Table
         return true;
     }
 
-    bool ensure_data(wxSQLite3Database* db)
-    {
-        try
-        {
-            db->ExecuteUpdate(wxString::Format("INSERT INTO SUBCATEGORY_V1 VALUES (1, '%s', 1)", wxTRANSLATE("Telephone")));
```

Comment:
Should be
db->ExecuteUpdate(wxString::Format("INSERT INTO SUBCATEGORY_V1 VALUES (1, '%s', 1)", _("Telephone")));